publish/samplesite/scripts/controls/ribbon/Ribbon.tsx

```diff
@@ -46,6 +50,25 @@ export default class Ribbon extends React.Component<RibbonProps, {}> {
         );
     }
 
+    private onRotateImage = () => {
+        const editor = this.props.plugin.getEditor();
+        blockFormat(editor, (region) => {
+            const blocks = getSelectedBlockElementsInRegion(region);
+            blocks.forEach(block => {
+                const blockElement = block.collapseToSingleElement();
+                for (let j = 0; j < blockElement.children.length; j++) {
+                    {
+                        const child = blockElement.children[j];
+                        if (child.tagName == "IMG") {
+                            rotateImage(editor, child as HTMLImageElement, 90);
+                            break; // Only rotate the first image

```

Comment:
Why do we want to rotate only the first image if we have multiple images selected?

Reply:
I just figured that since we are just trying to display how the behavior works that we didn't need to do all the selected images

Reply:
Yes, you just need to get current selected image, no need to get all.
You may need to do some change to ImageResize plugin to make the current selected image work for both places.

Reply:
Ive fixed this